### DISCUSSION TITLE
Chainlit UI integration with canopy server

### DISCUSSION THREAD

**Discussion by evanrsl** [pinecone-io/canopy#272] [Q&A]
After I run _canopy start_ , the server is now running on http://0.0.0.0:8000.
And I read in the README, it is said that i could use the server with chat application.

_That's it! you can now start using the Canopy server with any chat application that supports a /chat.completion endpoint._

I also see in the deployment-gcp.md, I can use any OpenAI compatible chat UI to interact with the newly deployed server.

Is there any example how to use the canopy server with a chat UI for example Chainlit?

Hope someone could help me on this, Thank You very much

**Comment by rene-schwabe** [comment]
Here you can see an example: https://github.com/Chainlit/cookbook/pull/84
